File: Modul 6/SOAL 4/Bahasa C/PRAK504-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>
int reverse(int angka){
    int sisa ;
    int kebalikan = 0 ;
    while (angka != 0){
        sisa = angka % 10 ;
        kebalikan  = kebalikan * 10 + sisa ;
        angka = angka / 10 ;}
    return kebalikan ;
}

int main () {
    int A, B;
    scanf("%d %d",&A,&B);
    A = reverse(A) ;
    B = reverse(B) ;
    int C = A + B ;
    printf("%d", reverse(C));
}

File: Modul 6/SOAL 4/Bahasa Python/PRAK504-2210817210028-Muhammad Aulia Rasyid.py
def reverse (angka) :
    kebalikan = 0
    while int(angka) != 0 :
        sisa = int(angka) % 10
        kebalikan = kebalikan * 10 + sisa
        angka = int(angka) / 10
    return kebalikan
A, B = input().split()
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))